src/dkg/round1.rs

```diff
@@ -239,6 +240,20 @@ impl PublicPackage {
     }
 }
 
+impl Ord for PublicPackage {
```

Comment:
@andreacorbellini mentioned [here](https://github.com/iron-fish/ironfish-frost/pull/53#discussion_r1560187532) that we can use `sort_unstable_by_key` for sorting

Reply:
Updated